README.md

```diff
@@ -0,0 +1,106 @@
+OpenIddict<br><sub>The OpenID Connect server you'll be addict to.</sub>
+=======
+
+AppVeyor: [![Build status](https://ci.appveyor.com/api/projects/status/d0d8git3o6lqkvbm?svg=true)](https://ci.appveyor.com/project/aspnet-contrib/core)
+
+Travis: [![Build status](https://travis-ci.org/openiddict/core.svg)](https://travis-ci.org/openiddict/core)
+
+### What's OpenIddict ?
+
+OpenIddict aims at providing a simple and easy out of the box solution 
+to implement an OpenID Connect server for ASP.NET 5.
+
+
+### Why an OpenID Connect Server?
+
+With an OpenID Connect Server you can manage all your users using local 
+password or an external identity provider management for all your applications 
+in one central place, with the power to control who can access your API and 
+the information that is exposed to each client. 
+
+
+### How does it work?
+
+OpenIddict, by default, leverages the use of Identity (for user management) and 
+EntityFramework (as an optional store provider).
+
+Adding it to your existing application allows you to register clients and 
+serve them authenticate tokens.
```

Comment:
`authenticate` tokens or `authenticaTION` tokens? (please don't throw stones at me if I've got this wrong, i always mess up my words...)